### PR TITLE
Add LianLi plugin link to README

### PR DESCRIPTION
**Pull request by EightB1ts**
Added a link to the LianLi plugin I put together. [https://github.com/EightB1ts/FanControl.LianLi](https://github.com/EightB1ts/FanControl.LianLi)

Needs some cleanup, but it's working. Some controllers need to be tested, but I was able to work with the L-Connect to determine the HID spec for all controllers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ From the community (notify me if I'm missing some):
 * https://github.com/EvanMulawski/FanControl.Razer to interface with Razer devices
 * https://github.com/hgross/FanControl.HomeAssistant to interface with [HomeAssistant](https://github.com/home-assistant) connected temperature sensors (i.e. ambient temperatures via Philips Hue, HomeMatic, HomeKit or many other brands & protocols)
 * https://github.com/brokenmass/Fancontrol.NzxtKraken to interface with NZXT Kraken AIO that are not yet supported by LibreHardwareMonitor for example `Kraken X2` and `Kraken X3 - new PID`. See [LHM PR](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/pull/1078)
+* https://github.com/EightB1ts/FanControl.LianLi to interface with LianLi fan controllers
```
Comment (Rem0o) [inline README.md:84]: Maybe mention (L-Connect) in the description.

**Comment by Rem0o** [comment]
Awesome!  

Got a few requests for L-Connect support over time, hope you get some traffic to develop this through. Feel free to look in the issues of this repo and link to your plugin if you want some beta-testers.

File: README.md (hunk 1 above)
Comment (EightB1ts) [reply README.md]: Updated in this commit: https://github.com/Rem0o/FanControl.Releases/pull/1829/commits/f2dfe539527c2585c834965bb8656df8018c6d00

**Comment by EightB1ts** [comment]
Thanks! I've resolved that issue you opened and updated the README to mention L-Connect.

**Pull request merged**